Platform/AlderlakeBoardPkg/Include/PlatformBoardId.h

```diff
@@ -75,4 +75,16 @@ Defines Platform BoardIds
 
 #define PLATFORM_ID_RPL_P_DDR5_CRB                    0x08
 
+// RPL-P Auto Malibou Lake Boards
+#define BoardIdRplPLp5AutoRvp                         0x05  // Standard 6 bits Board ID used when reading from IO Expander
+#define ExtendedBoardIdRplPLp5AutoRvp                 0x45  // MBL RVP Platform ID when read from FRU EEPROM
+#define BoardIdRplPAutoLp5Crb                         0x46  // MBL CRB Platform ID when read from FRU EEPROM
+
+#define PLATFORM_ID_RPLP_LP5_AUTO_RVP                 0x05  // Internal Platform Id for Malibou Lake RVP Fab. A and B.
+#define PLATFORM_ID_RPLP_LP5_AUTO_CRB                 0x06  // Internal Platform Id for Malibou Lake CRB.
```

Comment:
This conflicts with PLATFORM_ID_ADL_N_UP7EN50 (cf. line 22, above).
Any chance to get a unique value? (0x1..3, 0xc, 0xe..f, 0x15..16, 0x1a and 0x1d..1f seem to be available).

Reply:
Done.